src/components/clusterConfiguration/ClusterConfiguration.tsx

```diff
@@ -327,6 +308,28 @@ const ClusterConfiguration: React.FC<ClusterConfigurationProps> = ({ cluster })
                         isDisabled={!Object.keys(subnetHostMap).length}
                         validate={(value: string) => validateVIP(subnetHostMap, values, value)}
                       />
+                      <TextContent>
+                        <Text component="h2">Advanced Networking</Text>
+                      </TextContent>
+                      <InputField
+                        name="clusterNetworkCidr"
+                        label="Cluster Network CIDR"
+                        helperText="IP address block from which Pod IPs are allocated This block must not overlap with existing physical networks. These IP addresses are used for the Pod network, and if you need to access the Pods from an external network, configure load balancers and routers to manage the traffic."
+                        isRequired
```

Comment:
advanced fields are not required, right ?

Reply:
I believe they are required to be filled in, there will be defaults provided on cluster creation but if user removes that value the deployment won't proceed IIUC. @filanov ?

Reply:
I just got access to the creation form for OpenShift Dedicated clusters in OCM and noticed that they make these "advanced" fields optional but with placeholder text to show the default value that will be used if the user doesn't add anything else. What do you think of using the same pattern?

![image](https://user-images.githubusercontent.com/9122899/83462213-9ac5ce80-a438-11ea-9544-4b02b6ab109a.png)